docker/core/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM openjdk:8-slim
```

Comment:
Should we bump this to Java 11?

https://jaxenter.com/nobody-puts-java-container-139373.html
https://bugs.openjdk.java.net/browse/JDK-8146115

Reply:
Not sure if there are any compatibility issues, will test. 
Java 8 is the most ubiquitous so I'm not sure whether we should bump it or not if 11 is not backwards compatible

Reply:
Performance is ok with the additional flags

Reply:
Ok, let's make it a new issue if necessary

Reply:
Dataflow currently does not work if you compile it with java 11. Java 8 is recommended